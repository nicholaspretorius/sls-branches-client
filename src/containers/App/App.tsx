import React from "react";
import "./App.css";

import Header from "../../components/Header/Header";

const App: React.FC = () => {
  return (
    <div className="App container">
      <Header title="Branchly" />
      <header className="App-header">
        <p>Hello world!</p>
      </header>
    </div>
  );
};

export default App;
