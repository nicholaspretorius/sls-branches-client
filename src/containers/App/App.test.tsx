import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

describe("App", () => {
  test("renders learn react link", () => {
    const { getByText } = render(<App />);
    const linkElement = getByText("Hello world!");
    expect(linkElement).toBeInTheDocument();
  });

  it("renders", () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
