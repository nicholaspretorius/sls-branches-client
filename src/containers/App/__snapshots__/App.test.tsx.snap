// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <header
      class="App-header"
    >
      <p>
        Hello world!
      </p>
    </header>
  </div>
</DocumentFragment>
`;
